2019-11-10 02:14:03,172	INFO	main1	<module>	Line:11	app started
2019-11-10 02:14:03,190	WARNING	_internal	_log	Line:122	 * Debugger is active!
2019-11-10 02:14:03,236	INFO	_internal	_log	Line:122	 * Debugger PIN: 185-856-785
2019-11-10 02:14:03,256	INFO	_internal	_log	Line:122	 * Running on http://192.168.2.210:9100/ (Press CTRL+C to quit)
2019-11-10 02:14:08,572	INFO	_internal	_log	Line:122	192.168.2.210 - - [10/Nov/2019 02:14:08] "GET /ProblemSet/fa8624a7-de2a-4507-ba23-85a36f820eda/ww HTTP/1.1" 200 -
2019-11-10 02:14:09,646	INFO	_internal	_log	Line:122	192.168.2.210 - - [10/Nov/2019 02:14:09] "GET /ProblemSet/init/fa8624a7-de2a-4507-ba23-85a36f820eda/ww HTTP/1.1" 200 -
2019-11-10 02:14:09,682	INFO	_internal	_log	Line:122	192.168.2.210 - - [10/Nov/2019 02:14:09] "GET /ProblemSet/answer/fa8624a7-de2a-4507-ba23-85a36f820eda/1 HTTP/1.1" 500 -
2019-11-10 02:14:31,311	INFO	_internal	_log	Line:122	 * Detected change in 'C:\\development\\examsystem\\services\\daos\\problemsetDAO.py', reloading
